docs/getting_started/videoclips.rst

```diff
@@ -106,7 +106,7 @@ TextClip
 
 Generating a TextClip requires to have ImageMagick installed and (for windows users) linked to MoviePy, see the installation instructions.
 
-Here is how you make a textclip (you won't need all these options all the time): ::
+Here is how you make a text clip (you won't need all these options all the time): ::
```

Comment:
Another case of: would go with the spelling of MoviePy class names, i.e. correct to TextClip.